deps/verifier/src/snp/mod.rs

```diff
@@ -307,6 +310,38 @@ fn get_common_name(cert: &x509::X509) -> Result<String> {
     Ok(e.data().as_utf8()?.to_string())
 }
 
+// Function to request vcek from KDS. Return vcek in der format.
+fn request_endorsement_key_kds(
+    att_report: AttestationReport,
+) -> Result<Vec<CertTableEntry>, anyhow::Error> {
+    // KDS URL parameters
+    const KDS_CERT_SITE: &str = "https://kdsintf.amd.com";
+    const KDS_VCEK: &str = "/vcek/v1";
```

Comment:
might be better to put these consts at the top of the file rather than inside this function

Reply:
Fixed. If appropriate, please mark this comment as Resolved.